src/iam_roles/iamrolesservice.py

```diff
@@ -221,4 +246,4 @@ def lambda_handler(event, context):
         os.environ["prometheus_ip"], job="IAM-roles-service-data", registry=registry
     )
 
-    return {"statusCode": 200, "body": json.dumps("Service Lambda Data Pushed")}
+    return {"statusCode": 200, "body": json.dumps("Service Lambda Data Pushed")}
```

Comment:
Add EOF

Reply:
added in d4d700c790b5926a49486872d2b4441380703fa1